src/reynir_correct/main.py

```diff
@@ -335,14 +337,41 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
             )
             for ann in a
         ]
+        if args.text:
+            if sent.tree is None:
+                # No need to do more, no grammar errors have been checked
+                print(cleaned, file=args.outfile)
+            else:
+                # We know we have a sent tree, can use that
+                # Use toklist and a
+                cleantoklist: List[CorrectToken] = toklist
+                for xann in a:
+                    if not xann.suggest:
+                        # Nothing to correct with, nothing we can do
+                        continue
+                    if xann.start == xann.end:
+                        # Single-token grammar error, easy to correct
+                        if xann.suggest == "DELETE":
```

Comment:
Remove this branch of the if, keep the else branch

Reply:
Done